conference-finder/src/App.css

```diff
@@ -20,3 +20,18 @@
 .App-link {
   color: #09d3ac;
 }
+
+.card {
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s
+}
+
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+li {
```

Comment:
If you can, try have a read of the flex stuff we did the other day, it'll be better going forward as your items won't just scale down - the margins will shrink instead